runtime/shared_common/shrinit.cpp

```diff
@@ -3952,6 +3953,25 @@ j9shr_getCacheSizeBytes(J9JavaVM *vm)
 	return ret;
 }
 
+/**
+ * check if SCC is the default SSC (was enabled b/c of -Xshareclasses)
+ *
+ * @param [in] vm Pointer to the VM structure for the JVM
+ *
+ * @return The softmx bytes or the total usable cache size.
+ * 
+ * J9_ARE_ALL_BITS_SET(javaVM->sharedClassConfig->runtimeFlags, J9SHR_RUNTIMEFLAG_ENABLE_CACHE_NON_BOOT_CLASSES)
+ */
+BOOLEAN
+j9shr_isSCCDefault(J9JavaVM *vm)
+{
+	BOOLEAN ret = FALSE;
+	if (J9_ARE_ALL_BITS_SET(vm->sharedClassConfig->runtimeFlags, J9SHR_RUNTIMEFLAG_ENABLE_CACHE_NON_BOOT_CLASSES)){
```

Comment:
Please and a comment saying "Only bootstrap classes are shared by default."

Reply:
We should return `TRUE` if `J9SHR_RUNTIMEFLAG_ENABLE_CACHEBOOTCLASSES` is set but `J9SHR_RUNTIMEFLAG_ENABLE_CACHE_NON_BOOT_CLASSES` is not set.

Reply:
formatting: space before `{`